models/behavior_tests/rest_xml_protocol/2019-07-16/behavior.json

```diff
@@ -0,0 +1,2861 @@
+          "operation": "PutWithRestBindings",
+          "input": {
+            "UriPathSegment": "segment",
+            "UriPath": "path",
+            "HeaderMapWithPrefix": {
+              "Key-1": "prefix map value 1",
+              "Key-2": "prefix map value 2",
+              "Key-3": "prefix map value 3"
+            }
+          }
+        },
+        "expect": [
+          {
+            "requestHeadersMatch": {
+              "Prefix-Key-1": "prefix map value 1",
+              "Prefix-Key-2": "prefix map value 2",
+              "Prefix-Key-3": "prefix map value 3"
+            }
+          }
+        ]
+      },
+      {
+        "description": "Serializes payload blob members to the body as raw bytes",
+        "request": {
+          "operation": "PutWithRestBindings",
+          "input": {
+            "UriPathSegment": "segment",
+            "UriPath": "path",
+            "BodyStream": "YmluYXJ5LXZhbHVl"
+          }
+        },
+        "expect": [
+          {
+            "requestBodyEqualsBytes": "YmluYXJ5LXZhbHVl"
+          }
+        ]
+      },
+      {
+        "description": "Omits the HTTP body when payload blob member is not set",
+        "request": {
+          "operation": "PutWithRestBindings",
+          "input": {
+            "UriPathSegment": "segment",
+            "UriPath": "path"
+          }
+        },
+        "expect": [
+          {
+            "requestBodyEqualsString": ""
+          }
+        ]
+      },
+      {
+        "description": "Serializes payload structure members to the body as XML",
+        "request": {
+          "operation": "OperationWithPayloadStructureMember",
+          "input": {
+            "Data": {
+              "Value": "string value"
+            }
+          }
+        },
+        "expect": [
+          {
+            "requestBodyMatchesXml": "<DataNode xmlns=\"http://xml/ns\"><Value>string value</Value></DataNode>"
+          }
+        ]
+      },
+      {
+        "description": "Can serialize an empty structure to XML",
+        "request": {
+          "operation": "OperationWithPayloadStructureMember",
+          "input": {
+            "Data": {}
+          }
+        },
+        "expect": [
+          {
+            "requestBodyMatchesXml": "<DataNode xmlns=\"http://xml/ns\"/>"
+          }
+        ]
+      },
+      {
+        "description": "Can serialize an empty structure to XML",
+        "request": {
+          "operation": "OperationWithPayloadStructureMember",
+          "input": {
+            "Data": {}
+          }
+        },
+        "expect": [
+          {
+            "requestBodyMatchesXml": "<DataNode xmlns=\"http://xml/ns\"/>"
+          }
+        ]
+      },
+      {
+        "description": "Serializes members without locations to the body XML document",
+        "request": {
+          "operation": "OperationMembersWithoutLocation",
+          "input": {
+            "Header": "value 1",
+            "String_": "value 2",
```

Comment:
This field should not be changed to `String_`. the original name of `String` needs to be used in the behavior tests. the `_` is an artifact of the Go SDK, and won't translate to the other SDKs.  The `ShapeValueBuilder` should cross reference the member Shape reference name with the reference's original name.

Created #2806 to track the SDK's loss of this information.